docker/local-pytest/entrypoint.sh

```diff
@@ -1,6 +1,5 @@
-# Install PyCDS
-pip install -i https://pypi.pacificclimate.org/simple -r requirements.txt -r test_requirements.txt
-pip install -e .
+# Install pdp_util
+pipenv install
```

Comment:
This command will only install the packages from the `[packages]` section of the `Pipfile`. We will have to add the `--dev` flag if we want to get the test requirements as well.